### DISCUSSION TITLE
Question: how to close run-shell-command's floating output window?

### DISCUSSION THREAD

**Discussion by DuckDuckWhale** [helix-editor/helix#2944] [Q&A] [answered]
After running `:sh echo hi`, there is a floating `hi` on my window and I can't figure out how to close it. I couldn't find this in the docs either.

**Comment by the-mikedavis** [comment, marked as answer]
Escape or <kbd>Ctrl</kbd><kbd>c</kbd> will close any poppup. Each works for the popup from `:sh` as well as auto-complete (and signature help as well once merged).

**Reply by DuckDuckWhale** [reply]
Thanks. I'll ask future questions as discussions. I don't know why I didn't try escape.
